### ISSUE TITLE
Warning shown (not affecting the alerts)

### ISSUE THREAD

**Issue by thefabbulus** [SwiftKickMobile/SwiftMessages#435]
Version installed: 8.0.3 with SPM.
Xcode version: 12.0.

```
Unknown class _TtC27SwiftMessages_SwiftMessages18CornerRoundingView in Interface Builder file
```

I thought that it could be related to
```
(myView as? CornerRoundingView)?.cornerRadius = 5
```
but it appears even after changing it to
```
myView.layer.cornerRadius = 5
```

Thanks for the amazing job!

**Comment by wtmoose** [comment]
Can you try cleaning your derived data and restarting Xcode?

If the problem persists, how are you installing SwiftMessages, e.g. CocoaPods?

**Comment by tbaranes** [comment]
I have the same warning when installing `SwitMessages` using SPM on Xcode 12, even after cleaning derived data...

**Comment by thefabbulus** [comment]
hey @wtmoose! 
Installed with SPM, and already cleaned derived data a couple of times.
Still there.

**Comment by Dario-GoldenSpear** [comment]
Hi, 
I was having the same issue. I got rid of the warning by changing the module name for the `CorneredRoundingView` in IB to `SwiftMessages` instead of inheriting the module from the target, as shown here:
![CornerRoundingView-module-in-IB](https://user-images.githubusercontent.com/58740705/100259365-e60ff400-2f26-11eb-9496-3c28b2124887.png)

Makes sense as that class is not part of the app's target module but part of the `SwiftMessages` library. Although it was not affecting the expected behavior in runtime 🤔

**Comment by wtmoose** [comment]
This should be resolved in 8.0.4
